scientific/src/commonMain/kotlin/converter/area/convertToTime.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ Copyright 2021 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.scientific.converter.area
+
+import com.splendo.kaluga.scientific.PhysicalQuantity
+import com.splendo.kaluga.scientific.ScientificValue
+import com.splendo.kaluga.scientific.converter.time.time
+import com.splendo.kaluga.scientific.unit.Area
+import com.splendo.kaluga.scientific.unit.ImperialArea
+import com.splendo.kaluga.scientific.unit.ImperialKinematicViscosity
+import com.splendo.kaluga.scientific.unit.KinematicViscosity
+import com.splendo.kaluga.scientific.unit.MetricArea
+import com.splendo.kaluga.scientific.unit.MetricKinematicViscosity
+import kotlin.jvm.JvmName
+
+@JvmName("metricAreaDivMetricKinematicViscosity")
```

Comment:
Time is always in SI, so I dont think there's any reason to have special metric and imperial cases here. Just defaulting to the areaDivKinematicViscosity should yield the same result.

Reply:
good point, fixed